mitodl/ocw-studio
# type -> ocw_type

#### Pre-Flight checklist


#### What are the relevant tickets?
None

#### What's this PR do?
We had to update `type` to `ocw_type` in ocw-to-hugo output since type is a reserved keyword.

#### How should this be manually tested?
Currently this can't be tested since the pipeline to update `ocw-to-hugo-output-qa` is running. Once that's done

Run
`docker-compose run web ./manage.py overwrite_ocw_course_content --bucket ocw-to-hugo-output-qa --filter 6-007-electromagnetic-energy-from-motors-to-lasers-spring-2011 --content-field metadata.ocw_type`

Run
`docker-compose run web ./manage.py overwrite_ocw_course_content --bucket ocw-to-hugo-output-qa --filter 6-007-electromagnetic-energy-from-motors-to-lasers-spring-2011 --content-field metadata. learning_resource_types `

Verify that both work fine
